app/templates/admin/content/translations.hbs

```diff
@@ -16,7 +16,7 @@
               {{get l10n.availableLocales language.code}}
             </div>
             <div class="column eight wide">
-              <a href="{{language.sourceUrl}}">{{t 'Download'}}</a>
+              <button class="ui primary loading button" href="{{language.sourceUrl}}" {{action 'translationsDownload'}}>{{t 'Download' }}</button>
```

Comment:
Try the functionality yourself and see if it works before pushing .. if you add the loading class directly button will always be in the loading state.
Make it 
```hbs
<button class="ui primary {{if isLoading 'loading'}} button" href="{{language.sourceUrl}}" {{action 'translationsDownload'}}>{{t 'Download' }}</button>
```

Reply:
Yeah, made the requested changes. Also, for the downloads to work, made a PR to change the API endpoint to 'v1/admin/content/translations/all'